pandas/tests/reshape/test_pivot.py

```diff
@@ -2153,3 +2153,20 @@ def test_pivot_index_none(self):
 
         expected.columns.name = "columns"
         tm.assert_frame_equal(result, expected)
+
+    def test_pivot_index_list_values_none(self):
+        # Tests when index is a list and values None. See GH#37635
+        data = {
+            "index": ["A", "B", "C", "C", "B", "A"],
+            "columns": ["One", "One", "One", "Two", "Two", "Two"],
+            "values": [1.0, 2.0, 3.0, 3.0, 2.0, 1.0],
+        }
+        index = ["index"]
+        columns = ["columns"]
+        frame = DataFrame(data)
+        frame.pivot(index=index, columns=columns, values=None)
+
+        expected_ser = Series(["index"])
```

Comment:
Why are you converting to Series? you want to check that `index` itself (the list) did not change, no?

Reply:
Was trying to use the assert functions in `pandas._testing`.  Should I just use standard `assert`?

Reply:
yes use a standard assert here